### PR TITLE
Migrating "Business Logic" topics to Symfony Flex structure

### PR DESCRIPTION
**Pull request by yceruto**

### CODE REVIEW

File: best_practices/business-logic.rst [hunk 1]
```diff
@@ -10,48 +10,20 @@ your app that's not specific to the framework (e.g. routing and controllers).
 Domain classes, Doctrine entities and regular PHP classes that are used as
 services are good examples of business logic.
 
-For most projects, you should store everything inside the AppBundle.
+For most projects, you should store everything inside the ``src/`` directory.
```
Comment (javiereguiluz) [inline best_practices/business-logic.rst]: `store everything` -> `store all your code` ? Now almost everything is outside `src/` (translations, templates, assets, etc.)

**Review by javiereguiluz — APPROVED** [review]
In a separate PR, when we rethink the "best practices" we should reword this article to say that autowiring is a good practice and making services public to get them from the container is not a good practice.

File: best_practices/business-logic.rst (hunk 1 above)
Comment (yceruto) [reply best_practices/business-logic.rst]: Done, thanks!

**Comment by weaverryan** [comment]
Another great one! As Javier said, we need to read the best practices for bigger changes (updated recommendations). But updating it at least to be accurate to the new structure first makes a lot of sense :)

**Pull request merged**